elfpy/markets/hyperdrive/hyperdrive_market.py

```diff
@@ -256,11 +260,6 @@ def __init__(
         self.position_duration.freeze()  # pylint: disable=no-member # type: ignore
         super().__init__(pricing_model=pricing_model, market_state=market_state, block_time=block_time)
 
-    @property
```

Comment:
Why did you delete this?

Reply:
I saw it wasn't used anywhere and it's just returning a variable without any calculation

Reply:
It is being used in the bots_on_solidity notebook -- please leave it in.

Reply:
put it back in 7419cf280eaa85048f4c55732913aa1a4716630a